mockoon/docker-compose.mockoon.yml

```
@@ -0,0 +1,8 @@
+services:
+  mockoon:
+    build:
+      context: .
+      dockerfile: Mockoon.Dockerfile
+    ports:
+      - "8080:8080"
+    restart: unless-stopped
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)